packages/statistics-module/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "statistics-module",
+  "version": "1.0.0",
+  "description": "> TODO: description",
+  "author": "Rubén Esparza <rubenesparzasoto95@gmail.com>",
```

Comment:
We should configure the author property with the name of the team that owns the service.

Reply:
We will change that and the description 📝